ctl/src/command.rs

```diff
@@ -1306,10 +1311,11 @@ pub fn query_application(mut channel: Channel<CommandRequest,CommandResponse>, j
           // do nothing here
           // for other messages, we would loop over read_message
           // until an error or ok message was sent
+          Ok(())
         },
         CommandStatus::Error => {
           if json {
-            print_json_response(&message.message);
+            print_json_response(&message.message)?;
           } else {
             eprintln!("could not query proxy state: {}", message.message);
```

Comment:
Is it intentional to not bail here? And keep the exit(1) below? If not, don't forget to return an error above :)

Reply:
I have forgotten this spot, thanks.